lifetiles-sequence/src/main/java/nl/tudelft/lifetiles/sequence/controller/SequenceController.java

```diff
@@ -102,30 +97,35 @@ public void initialize(final URL location, final ResourceBundle resources) {
         registerShoutListeners();
         initializeTable();
 
-        visibilityListeners = new HashMap<>();
+        visibleListeners = new HashMap<>();
     }
 
     /**
      * Register the shout listeners.
      */
+    // checkstyle bug causes false positives in our assert, so suppress.
+    @SuppressWarnings("checkstyle:genericwhitespace")
     private void registerShoutListeners() {
-        listen(Message.LOADED,
-                (sender, subject, args) -> {
-                    if (!subject.equals("sequences")) {
-                        return;
-                    }
-                    assert (args[0] instanceof Map<?, ?>);
+        listen(Message.LOADED, (sender, subject, args) -> {
+            if (!"sequences".equals(subject)) {
+                return;
+            }
+            // eclipse and PMD disagree on whether parentheses are
+            // needed
+                assert args[0] instanceof Map<?, ?>;
```

Comment:
Formatter does this. If I manually change it then the next time someone modifies this file and runs the formatter(as they should), it'll be reverted again.